model/src/main/kotlin/licenses/ResolvedLicenseFileInfo.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.licenses
+
+import java.io.File
+
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Provenance
+
+/**
+ * Information about license files of a package and the licenses detected in those files.
+ */
+class ResolvedLicenseFileInfo(
+    /**
+     * The identifier of the package.
+     */
+    val id: Identifier,
+
+    /**
+     * The resolved license files.
+     */
+    val files: List<ResolvedLicenseFile>
+)
+
+/**
+ * Information about a single resolved license file.
+ */
+class ResolvedLicenseFile(
+    /**
+     * The [Provenance] of the license file.
+     */
+    val provenance: Provenance,
+
+    /**
+     * The [ResolvedLicense]s detected in this file. Contains the whole resolved license including license and copyright
+     * locations found in other files with the same [provenance].
+     */
+    val licenses: List<ResolvedLicense>,
+
+    /**
+     *  The path of the license file relative to the [provenance].
+     */
+    val path: String,
+
+    private val file: File
```

Comment:
Since this is a constructor argument, could you please add docs for it, too?

Reply:
Done. Also made it public.